src/rbac.rs

```diff
@@ -31,14 +34,32 @@ use crate::state::workload::{WorkloadError, byte_to_ip};
 use crate::strng::Strng;
 use crate::{strng, xds};
 
-#[derive(Debug, Hash, Eq, PartialEq, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct Authorization {
     pub name: Strng,
     pub namespace: Strng,
     pub scope: RbacScope,
     pub action: RbacAction,
     pub rules: Vec<Vec<Vec<RbacMatch>>>,
+    #[serde(default)]
+    pub stats: Arc<AuthzCounter>,
+}
+
+#[derive(Debug, Default, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct AuthzCounter {
+    hits: AtomicU64,
+    misses: AtomicU64,
+}
```

Comment:
hit and miss are relative to the kind of policy. hit a deny == denied. hit an allow == allowed. Would users be better served with "allows" and "denies" which inform them of a concrete result taken in association with this policy?

Reply:
I worry an allow policy with `deny: 123` looks alarming. Its not really a deny, its more like it failed to allow

Reply:
So if a policy doesn't match, `miss`/`deny` will be incremented, even if another policy later on allow or denies it?

Reply:
I also wonder if the order of operations might be confusing cause the stats to be confusing. For example, if there are multiple deny policies, but only one of them matches, then the order in which the deny policies are evaluated will affect the stats (assuming there is some short circuiting), which seems weird.

Reply:
Maybe a better say of saying this is, why do we need `miss` when we can just do `total_requests - hit`

Reply:
there is no total request counter for a policy

Reply:
I mean total request counter across for ztunnel in general

Reply:
That doesn't exist is the first issue. We could add it but it doesn't help the use case. 
Miss means "policy applies to the request but does not match"

Reply:
Ok then only my second comment then: This doesn't seem to take into account short circuiting. Just because a policy could apply, short circuiting might mean that a counter doesn't get incremented, which might be the intended behavior